documentation/Releasing.md

```diff
@@ -1,6 +1,16 @@
-# Releasing in the Monorepo
+# Releasing
 
-The process for releasing the monorepo's contents is in active development. This document captures the current process for each piece.
+Polaris uses Changsets to handle releasing the following npm packages:
+
+- [polaris-icons](/polaris-icons)
+- [polaris-react](/polaris-react)
+- [polaris-tokens](/polaris-tokens)
+- [stylelint-polaris](/stylelint-polaris)
```

Comment:
```suggestion
Polaris uses Changsets to handle releasing the npm packages in repository.
```

I think focusing this on releasing NPM packages makes it easier to add more NPM packages and not worry about docs.